Library/Homebrew/os/mac/sdk.rb

```diff
@@ -0,0 +1,57 @@
+require "os/mac/version"
+
+module OS
+  module Mac
+    class SDK
+      attr_reader :version, :path
+
+      def initialize version, path
+        @version = OS::Mac::Version.new version
```

Comment:
Because `OS::Mac` is the actual module name and `MacOS` [is only there for compatibility](https://github.com/Homebrew/homebrew/blob/2d6637d800b7d1fe0e86c42296ced19e1d9e1382/Library/Homebrew/os/mac.rb#L11).